slobrok/src/tests/oldapi/old.cpp

```diff
@@ -136,10 +132,9 @@ Test::Main()
 
     std::vector<std::string> slobrokSpecs;
     slobrokSpecs.push_back("tcp/localhost:18531");
-    FRT_Supervisor orb;
-    MirrorOld mirror(orb, slobrokSpecs);
+    fnet::frt::StandaloneFRT server;
+    MirrorOld mirror(server.supervisor(), slobrokSpecs);
```

Comment:
possibly unstable test since mirror will become ready

Reply:
BTW, way do we both a 'current' and an 'old' API. Could the old one be GC'ed.

Reply:
Fixed